test/unit/puppetlabs/services/file_serving/config/puppet_fileserver_config_core_test.clj

```diff
@@ -0,0 +1,21 @@
+(ns puppetlabs.services.file-serving.config.puppet-fileserver-config-core-test
+  (:require [clojure.test :refer :all]
+            [puppetlabs.services.file-serving.config.puppet-fileserver-config-core :refer :all]))
+
```

Comment:
Probably should add

```clojure
(use-fixtures :once schema.test/validate-schemas)
```
here, to ensure that, at least during tests, the schemas are being validated.